.github/workflows/test-tf-plan.yml

```diff
@@ -0,0 +1,55 @@
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  find-changed-dirs:
```

Comment:
for projects that were really large (like shared-infra), this might not work. We might have to have static jobs or something (I think there is a limit on the number of matrix items you can have)

Reply:
We need to break shared-infra up.